[2022-08-13 12:28:29,386] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-13T12:28:25.753563+00:00 [queued]>
[2022-08-13 12:28:29,403] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-13T12:28:25.753563+00:00 [queued]>
[2022-08-13 12:28:29,404] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 12:28:29,405] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-13 12:28:29,405] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 12:28:29,426] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): download_raw_data> on 2022-08-13 12:28:25.753563+00:00
[2022-08-13 12:28:29,432] {standard_task_runner.py:52} INFO - Started process 4751 to run task
[2022-08-13 12:28:29,436] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'download_raw_data', 'manual__2022-08-13T12:28:25.753563+00:00', '--job-id', '364', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpkoemij42', '--error-file', '/tmp/tmplnm6c4ok']
[2022-08-13 12:28:29,439] {standard_task_runner.py:80} INFO - Job 364: Subtask download_raw_data
[2022-08-13 12:28:29,508] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.download_raw_data manual__2022-08-13T12:28:25.753563+00:00 [running]> on host 4bbb34367c41
[2022-08-13 12:28:29,602] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=download_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-13T12:28:25.753563+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-13T12:28:25.753563+00:00
[2022-08-13 12:28:29,604] {ETL_Raizen.py:40} INFO - Downloading .xls raw data
[2022-08-13 12:28:31,663] {ETL_Raizen.py:42} INFO - Succesfully downloaded .xls raw data
[2022-08-13 12:28:31,664] {python.py:173} INFO - Done. Returned value was: None
[2022-08-13 12:28:31,679] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=download_raw_data, execution_date=20220813T122825, start_date=20220813T122829, end_date=20220813T122831
[2022-08-13 12:28:31,701] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-13 12:28:31,752] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
